src/utils.c

```diff
@@ -139,3 +140,13 @@ int readFileNumber(long *value, const char *filename_fmt, ...)
   *value = n;
   return 0;
 }
+
```

Comment:
can this be moved to p2p_plugin.c and remove "p2p_plugin.h" in this file. keep this file for utils only